requirements.txt

```diff
@@ -0,0 +1,29 @@
+boto==2.40.0
+colander==1.3.1
+colorama==0.3.7
+cornice==1.2.1
+enum34==1.1.6
+functools32==3.2.3.post2
+iso8601==0.1.11
+jsonschema==2.5.1
+kinto==3.1.1
+PasteDeploy==1.5.2
+pyramid==1.7
```

Comment:
I am a bit worried about all this tagging - because you might get into problems if Kinto tags different version. Since pyramid is an indirect dependency, I would be inclined to just keep kinto==3.1.1 and let pip do the work when grabbing kinto